mobile-experimental/Yote/js/global/styles/tailwind/YTTailwind.js

```diff
@@ -0,0 +1,62 @@
+import React, { PropTypes } from 'react';
+
+import {
+  Dimensions
+  , Platform
+  , StyleSheet
+} from 'react-native'; 
+
+// import theme
+import theme from './themes/default'; 
+// import theme from '../themes/custom'; 
+
+import { tailwind, getColor } from './styles/tailwind'; 
+
+const { height, width } = Dimensions.get('window');
+
+const FONT = Platform.OS === 'android' ? 'sans-serif-condensed' : 'AvenirNextCondensed-DemiBold';
+
+const YTStyles = StyleSheet.flatten({
+
+  // layout
+  brandContainer: tailwind(`flex-1 bg-blue-500`)
+  , cell: tailwind('px-5 py-5')
+  , container: tailwind('flex-1 bg-white')
+  , logo: tailwind('w-60 h-60')
+  , icon: tailwind('w-20 h-20')
+  , inputContainer: tailwind('border-b border-transparent border-gray-100 m-10')
+  , separator: tailwind('border-t border-gray-100')
+  , userImg: tailwind('h-40 w-40 rounded-full justify-center')
+  , horizCenter: tailwind('justify-center items-center')
+
+  // text
+  , h1: tailwind('text-4xl font-semibold text-black')
```

Comment:
this will need circling back to at some point, not sure how erik wants to do these on mobile.